### PR TITLE
Task 147753699

### PR DESCRIPTION
**Pull request by gxa**

### CODE REVIEW

File: sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        result.addProperty("accessKey", accessKey);
+        result.addProperty("species", experiment.getSpecies().getReferenceName());
+        result.addProperty("disclaimer", experiment.getDisclaimer());
+
+        JsonArray availableTabs = new JsonArray();
+
+        availableTabs.add(tSnePlotTab(availableDataUnits(experiment.getAccession(), experiment.getType())));
+
+        availableTabs.add(customContentTab("none", "Experiment Design", new JsonObject()));
+//        if(dataFileHub.getExperimentFiles(experiment.getAccession()).experimentDesign.exists()){
```
Comment (wbazant) [inline sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java]: Time to remove the comments?

File: sc/src/main/javascript/build-bundles.sh [hunk 2]
```diff
@@ -18,9 +18,13 @@ do
     pushd . > /dev/null
     echo "Upgrading $dir:"
     cd $dir
-    yarn upgrade
+    #yarn upgrade
+    ncu -a
```
Comment (wbazant) [inline sc/src/main/javascript/build-bundles.sh]: which ncu? I don't have that

Comment (alfonsomunozpomer) [reply sc/src/main/javascript/build-bundles.sh]: I added a note in the commit message, just do `npm install -g npm-check-updates`. It’s quite useful. In my mind it’s what I would like `npm update`/`yarn upgrade` to do.

File: sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html [hunk 3]
```diff
@@ -14,28 +14,58 @@
...
+        name: 'Supplementary Information',
+        props: {}
+      },
+      {
+        type: '',
+        name: 'Downloads',
+        props: {}
+      }
+    ]
+
```
Comment (elisabetb) [inline sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html]: Just one question, are we going to have the same tabs as in gxa?

Comment (wbazant) [reply sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html]: I think the code for the tabs got copy pasted but probably the actual tabs will be different (maybe there won't be "Supplementary Information" or whatever)

File: sc/src/main/javascript/bundles/experiment-page/src/ExperimentPageRouter.js [hunk 4]
```diff
@@ -1,29 +1,116 @@
...
+  // Pass in the search from location
+  const Tab = tabTypeComponent[type]
+
+  return (
+    Tab ? <Tab {...tabProps} {...commonProps} {...routeProps}/> : null
+  )
+}
+
+TabContent.propTypes = {
+  type: PropTypes.string.isRequired,
```
Comment (elisabetb) [inline sc/src/main/javascript/bundles/experiment-page/src/ExperimentPageRouter.js]: What is type here?

File: sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html (hunk 3 above)
Comment (gxa) [reply sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html]: I guess that *Supplementary information* and *Experiment design* will be there, but the content will be differ substantially. At thte moment we have some empty placeholders. The current system of sending the tabs from the server and let the client side do the rendering is pretty flexible. This is what we agreed in the [Pivotal ticket](https://www.pivotaltracker.com/story/show/147753699) to have something very minimal.

Comment (alfonsomunozpomer) [reply sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html]: I guess that *Supplementary information* and *Experiment design* will be there, but the content will be differ substantially. At thte moment we have some empty placeholders. The current system of sending the tabs from the server and let the client side do the rendering is pretty flexible. This is what we agreed in the [Pivotal ticket](https://www.pivotaltracker.com/story/show/147753699) in order to have a minimal working implementation.

Comment (wbazant) [reply sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html]: You could even add the single-cell-specific tabs to the experiment page code (if you need rendering for all the other components and it's faster to reuse) and keep the same architecture.

Comment (alfonsomunozpomer) [reply sc/src/main/javascript/bundles/experiment-page/gxa_sc/experiments/E-MTAB-4388/index.html]: 👍

File: sc/src/main/javascript/bundles/experiment-page/src/ExperimentPageRouter.js (hunk 4 above)
Comment (alfonsomunozpomer) [reply sc/src/main/javascript/bundles/experiment-page/src/ExperimentPageRouter.js]: The type of the tab, it determines what the component that will be rendered in it. See https://github.com/gxa/atlas-experiment/blob/master/src/ExperimentContainer.js#L20 and `createPageSection` for a more complete example.

**Pull request merged**